### PR TITLE
bs4 update admin dropdowns

### PR DESCRIPTION
**Pull request by charlienparker**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [x] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [x] Any endpoint modified in the PR only responds to the expected MIME types.
- [x] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [x] There are no inline styles added
- [x] There are no inline javascript added
- [x] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [x] Code does not use .html_safe
- [ ] All images added/updated have alt text
- [x] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: 
1. https://www.pivotaltracker.com/story/show/187977134
2. https://www.pivotaltracker.com/story/show/187498358
3. https://www.pivotaltracker.com/story/show/187498359
4. https://www.pivotaltracker.com/story/show/187498360
5. https://www.pivotaltracker.com/story/show/187498362
6. https://www.pivotaltracker.com/story/show/187498363
7. https://www.pivotaltracker.com/story/show/187498364
8. https://www.pivotaltracker.com/story/show/187498365
9. https://www.pivotaltracker.com/story/show/187498369
10. https://www.pivotaltracker.com/story/show/187498366
11. https://www.pivotaltracker.com/story/show/187498357
12. https://www.pivotaltracker.com/story/show/187498352

# A brief description of the changes

Current behavior: Admin dropdowns are using legacy styles.

New behavior: Admin dropdowns use new styles and are functional with the admin flag on.

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name: BS4_ADMIN_FLOW_IS_ENABLED

- [ ] DC
- [x] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

### CODE REVIEW

File: app/javascript/css/forms.scss [hunk 1]
```diff
@@ -13,6 +13,10 @@
     }
   }
 }
+.form-inline label {
```
Comment (charlienparker) [inline app/javascript/css/forms.scss:16]: I think this can be removed now

**Review by kristinmerbach — DISMISSED** [review]
LGTM, looks good on screen when i tested locally

**Review by kristinmerbach — DISMISSED** [review]
LGTM

**Review by TristanB17 — DISMISSED** [review]

**Review by kristinmerbach — DISMISSED** [review]
LGTM

**Review by TristanB17 — DISMISSED** [review]

**Review by kristinmerbach — DISMISSED** [review]

**Review by kristinmerbach — APPROVED** [review]
LGTM

**Review by TristanB17 — APPROVED** [review]

**Review by kristinmerbach — APPROVED** [review]

**Pull request merged**